### PR TITLE
fix: pm app fixes

### PR DESCRIPTION
**Pull request by davidruvolo51**
What are the main changes you did:
- fixed calculations: in the project planning view, I adjusted the formulas to prevent a issue where calculations were larger than they actually are.
- Added free text field: added notes to the person planning view
- Added field to projectmanagement schema template
- HTML/CSS fixes:
    - Table header: multispan columns are properly marked up
    - "Nested rows" were structured to ensure cell/header links
    - Table header is now sticky
    - Adjusted border styling to align with multispan column headers
    - Adjusted padding throughout the table to increase the readability of the table

how to test:
- Log in to the preview
- Create a new schema using the "PROJECTMANAGER" template and click "load demo data"
- Navigate to the new schema and view the "project planning" and "persons planning" pages
- Add new records or update existing the demo data 

todo:

### CODE REVIEW

**Comment by mswertz** [comment]
+button doesn't work in my case
image

**Review by mswertz — CHANGES_REQUESTED** [review]
+ button didn't work in my case
image

**Comment by mswertz** [comment]
> * button didn't work in my case
> 
might be problem elsewhere.

**Comment by davidruvolo51** [comment]
It is unclear where the issue is. There is no error, warning, or event when add button is clicked. I think there may be an issue with the import of the `EditModal` in the [RowButtonEdit component at line 25](https://github.com/molgenis/molgenis-emx2/blob/d06e651e868c25a2fc49c0e0dc1959c801b6c565/apps/molgenis-components/src/components/tables/RowButtonEdit.vue#L25). When running locally, I'm seeing a lot of "unresolved import" warnings. Declaring the import in the RowButtonEdit component causes the build of `molgenis-components` to fail. Perhaps that is causing the issue, but it's really hard to tell.

**Comment by davidruvolo51** [comment]
Fixed import `EditModal` in the `RowButtonAdd` component

File: apps/molgenis-components/src/components/tables/RowButtonAdd.vue [hunk 1]
```diff
@@ -22,7 +22,11 @@
 <script setup lang="ts">
 import RowButton from "./RowButton.vue";
 import ButtonOutline from "../forms/ButtonOutline.vue";
-import { ref } from "vue";
+import { defineAsyncComponent, ref } from "vue";
+
+const EditModal = defineAsyncComponent({
+  loader: () => import("../forms/EditModal.vue"),
```
Comment (mswertz) [inline apps/molgenis-components/src/components/tables/RowButtonAdd.vue:28]: ah, right

**Review by mswertz — APPROVED** [review]

**Pull request merged**